addons/mail/static/src/messaging/component/thread_viewer/thread_viewer.js

```diff
@@ -183,6 +184,7 @@ Object.assign(ThreadViewer, {
             type: String,
             validate: prop => ['asc', 'desc'].includes(prop),
         },
+        scrollAdjust: Boolean,
```

Comment:
`optional` prop, since it forwards to `MessageList`.
`MessageList` itself should have a props similarly named (e.g. `hasScrollAdjust`), and it must define a default on this.